### ISSUE TITLE
Provide operational support for Marketing – Ongoing

### ISSUE THREAD

**Issue by kitblake** [rchain/bounties#816]
## Provide operational support for Marketing – Ongoing

The Marketing organization is in an embryonic phase and would like someone to help guide the setup. This includes tasks such as:

- Setup a Jira instance for project manegement with content structure
- Act as infrastructural liaison for the blog and future site projects
- Play a guide/scribe role in meetings
- Track planned activities
- Quality control
- Other tasks yet to be determined.

This issue is a request from Marketing and they would like @kitblake to take on the role. This would allow him to be a Sociocratic circle link with other Co-op entities.

### Budget and Objective

_Estimated Budget of Task:_ **$640/week** (actual hours will tracked)
_Estimated Timeline Required to Complete the Task:_ **1 month**
_Measure of Completion:_ ** Timesheet has been approved **

--
***Legal***
*Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws.*

**Comment by deannald** [comment]
Sounds like you guys need a Scrum Master :)

**Comment by philipandri002** [comment]
@kitblake @deannald.

Great idea but permit me add my little opinion. Maybe there should be a kind of seminar probably a webiner to help train, inspire and educated the marketing forum of the Rchain ccop.

No cooperation or cooperative can make the kind of impact without regular training which will serve as a guide and also as an inspiration to team member especially to the marketing team.

There has no be a method of evaluating workforce and improving the system from feedbacks from the market.

This will in no little way improve the marketing forum and increase the ROI and our capacity.

Thanks

**Comment by kitblake** [comment]
@philipandri002 Good point. Up till now the group has worked as a meritocracy where people who clearly demonstrate skills take on appropriate roles. But indeed, with recruitment it'll be necessary to have trainings where the master/journeyer people pass on their knowledge to apprentices.

**Comment by AyAyRon-P** [comment]
Hi @kitblake 

Are there any updates? Do we need to vote on this for the month of August? Let me know what your thoughts are on budget for the month so we can vote accordingly.

**Comment by dckc** [comment]
Oh... so this is what we were talking about today in #995
